mobile/src/screens/HomeScreen.tsx

```diff
@@ -1,85 +1,147 @@
-import React, { useContext, useEffect } from 'react';
-import { StyleSheet, Text, View, FlatList } from 'react-native';
+import React, { useContext, useEffect, useState } from 'react';
+import { StyleSheet, Text, View, ScrollView, KeyboardAvoidingView, Platform, Dimensions } from 'react-native';
+
 import { Svg, Circle } from 'react-native-svg';
 
-import { ColumnBookList } from '../components/ColumnBookList';
+import { BookFilter } from '../components/BookFilter';
+import { PaginatedBookList } from '../components/PaginatedBookList';
 import { HorizontalBookList } from '../components/HorizontalBookList';
+import { LoadingCircle } from '../components/LoadingCircle';
 import { BookContext } from '../context/BookContext';
 import { TextStyles } from '../styles/TextStyles';
 import { Colors } from '../styles/Colors';
 import { I18nContext } from '../context/I18nContext';
-import { LoadingCircle } from '../components/LoadingCircle';
+import { Language } from '../models/Languages';
+import { Book } from '../models/Book';
+
+// how many books to display per page (in the Paginated Booklist)
+const booksPerPage = 9;
+
+const { width } = Dimensions.get('window');
 
 /**
- * Renders the homescreen for the app. Currently displays heading, new books, all books.
- */
+* Renders the homescreen for the app. Currently displays heading, new books, all books.
+*/
```

Comment:
Did you override the indentation settings in your editor or something? Do you mind cleaning up all the small indentation changes?

Reply:
I think it was because the virtualizedView, which was unindendented was replaced with the scrollView, which was indented, so all its children got another space of indentation.